packages/dx-react-grid-demos/src/demo-sources/grid-tree-data/remote.jsxt

```diff
@@ -60,7 +56,11 @@ export default () => {
     }
   };
 
-  useEffect(() => loadData());
+  useEffect(() => {
+    if (!loading) {
```

Comment:
Why you use `!loading` here and only `loading` in the other demos?

Reply:
Because in `loadData` was condition:
`if (loading) {	
      return;	
    }`
Replace that condition in `useEffect` prevent excess render.